src/Pulp/Publish.purs

```diff
@@ -331,6 +331,7 @@ uploadPursuitDocs out authToken gzippedJson = do
     other -> do
       out.err =<< concatStream (HTTP.responseAsStream res)
       out.err $ HTTP.statusMessage res
+      out.bolded $ "If you are publishing a package and running this command for the first time, you will almost always need to run this same command a second time before your package gets published."
```

Comment:
```suggestion
      out.bolded $ "This command may fail with a 400 error from Pursuit on the first run. Try running it a second time before debugging further."
```

What do you think of this rewording that specifies what the error is?

Reply:
Works for me!